lang/en.lyaml

```diff
@@ -529,6 +529,7 @@ en:
             noAccountsInConfig: "No accounts found in your config. Run {{ authCommand }} to configure a HubSpot account with the CLI."
             invalidProjectComponents: "Projects cannot contain both private and public apps. Move your apps to separate projects before attempting local development."
             noRunnableComponents: "No supported components were found in this project. Run {{ command }} to see a list of available components and add one to your project."
+            invalidUnifiedAppsAccount: "You must target a Combined account to use Unified Apps Local Dev"
```

Comment:
I know I initially wrote this, but I think we want to use different terminology here. This should probably be something like:
"Cannot start local dev. Target a standard HubSpot account or a Developer account and try again."

Reply:
Updated it to require a developer account only, since we require you to create a dev test account and you can't do that with a standard account at the moment